osu.Game/Overlays/Profile/Sections/Historical/PaginatedMostPlayedBeatmapContainer.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System.Linq;
+using osu.Framework.Configuration;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Online.API.Requests;
+using osu.Game.Users;
+
+namespace osu.Game.Overlays.Profile.Sections.Historical
+{
+    public class PaginatedMostPlayedBeatmapContainer : PaginatedContainer
+    {
+        public PaginatedMostPlayedBeatmapContainer(Bindable<User> user)
+            :base(user, "Most Played Beatmaps", "No performance records. :(")

```

Comment:
wrong string

Reply:
![https://osu.ppy.sh/users/3](https://user-images.githubusercontent.com/12570757/33244314-51b5de36-d2f5-11e7-86d9-b99fa9587640.png)

Reply:
It shouldn't say "No performance records" since these aren't performance-related.